### ISSUE TITLE
Bulkrax #903

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/hykuup_knapsack#136]
https://github.com/samvera/bulkrax/issues/903

**Comment by ShanaLMoore** [comment]
I think that bulkrax_identifier indexing should be removed from Hyku. 

it's an optional value and each client will be set up differently.

**Comment by ShanaLMoore** [comment]
to continue: [slack convo](https://assaydepot.slack.com/archives/C0313NKG2DA/p1706563424640579) 
**tldr; we decided to remove references for now**

the [find_by_bulkrax_identifier.rb](https://github.com/WGBH-MLA/ams/blob/develop/app/services/hyrax/custom_queries/find_by_bulkrax_identifier.rb) files were brought over because gbh has it and is using bulkrax_identifier in their [field mapping.](https://github.com/WGBH-MLA/ams/blob/develop/config/initializers/bulkrax.rb#L54)

This is an[ optional](https://github.com/samvera/bulkrax/wiki/Configuring-Bulkrax) and [configurable](https://github.com/samvera/bulkrax/blob/main/lib/generators/bulkrax/templates/config/initializers/bulkrax.rb#L49-L51) property though. 

For example, Adventist doesn't have bulkrax_identifier and instead uses their [identifier as the source_identifier](https://github.com/scientist-softserv/adventist-dl/blob/main/config/initializers/bulkrax.rb#L41). I don't believe [default bulkrax](https://github.com/samvera/bulkrax/blob/main/lib/generators/bulkrax/templates/config/initializers/bulkrax.rb#L49-L51) has anything defined for source_identifier so this is something each knapsack should decide/set up (in my opinion)

It's included in Hyku's indexer because we brought over partial implementation from [pals](https://github.com/scientist-softserv/palni-palci/blob/main/app/indexers/app_indexer.rb#L18).

I know we talked about this before, but asking again, can't we just remove the partial implementation of bulkrax_identifier from hyku and worry about it later when we set up client's knapsack?

I'd argue this would also remove the need for the two find_by_bulkrax_identifier services
